### PR TITLE
rf69 driver enhancements

### PR DESCRIPTION
**Pull request by tve**
This PR has changes and enhancements to the RF69 driver. I didn't include my RF-level; tweaks (different modulation index, etc).

### CODE REVIEW

File: explore/1608-forth/flib/spi/rf69.fs [hunk 1]
```diff
@@ -7,12 +7,11 @@
        $11 constant RF:PA
        $18 constant RF:LNA
        $1F constant RF:AFC
-       $21 constant RF:FEI
```
Comment (tve) [inline explore/1608-forth/flib/spi/rf69.fs]: I can't get the FEI value to be meaningful.

File: explore/1608-forth/flib/spi/rf69.fs [hunk 2]
```diff
@@ -63,10 +62,11 @@ hex
...
   2D05 h, \ Preamble 5 bytes
-  2E88 h, \ sync size 2 bytes
-  2F2D h, \ sync1: 0x2D
-  302A h, \ sync2: network group
-  37D0 h, \ drop pkt if CRC fails
+  2E90 h, \ sync size 3 bytes
+  2FAA h, \ sync1: 0xAA
+  302D h, \ sync2: 0x2D
+  312A h, \ sync3: network group
+  37D0 h, \ drop pkt if CRC fails \ 37D8 h, \ deliver even if CRC fails
```
Comment (tve) [inline explore/1608-forth/flib/spi/rf69.fs]: I changed to 3 bytes of sync to include the last by of preamble. This was originally a John O improvement.

File: docs/flib/rf69.md [hunk 3]
```diff
@@ -12,21 +12,59 @@ find out whether a new packet has been received.
...
 
-[defs]: <> (rf-init rf-power rf-sleep)
+The driver and chip must be initialized by calling `rf-init` after setting
+the `rf.freq` and `rf.group` variables. `rf-sleep` can be used to put the
+radio into low-power sleep mode. To wake it up call `rf-recv` or `rf-send`.
+
+[defs]: <> (rf-init rf-sleep)
 ```
 : rf-init ( -- )  \ init RFM69 with current rf.group and rf.freq values
-: rf-power ( n -- )  \ change TX power level (0..31)
```
Comment (jcw) [inline docs/flib/rf69.md]: `rf-power` intentionally dropped from the docs?

Comment (tve) [inline docs/flib/rf69.md]: I moved it down with the freq correction. I thought I was going to add the power-adjustment words there too, but they may be too much. I guess I should move it back up.

**Pull request merged**